data/strings/types_strings.txt

```diff
@@ -12598,6 +12598,7 @@
   [type.junction]
     en = Junction
     ar = تقاطع
+    de = Straßenknotenpunkt
```

Comment:
I don't know what a "Straßenknotenpunkt" could possibly be if it's not an ordinary "Kreutzung".

Reply:
[Wiki](https://wiki.openstreetmap.org/wiki/DE%3AKey%3Ajunction) says it's a Straßeknotenpunkt. 'junction=yes' is not only used for crossroads but also t- and y-interesections. (See [here](https://wiki.openstreetmap.org/wiki/Tag:junction%3Dyes)) That's why 'Kreuzung' would be wrong here, I think.

Reply:
I don't know where this string is used so maybe I just don't get it, but I can't imagine "Straßenknotenpunkt" anywhere in the app to sound natural to anyone.

DE:Key:junction=*  ->  "Beschreibt die Art eines Straßenknotenpunkts"  (e.g. "Kreisverkehr", "T-Junction", "Y-Junction", etc.)

DE:Tag:junction=yes  ->  "Eine Straßenkreuzung als solche"  (i.e. "Kreutzung")

Reply:
> DE:Tag:junction=yes -> "Eine Straßenkreuzung als solche" (i.e. "Kreutzung")

I can't find this quote in german wiki... It says "junction=[yes](https://wiki.openstreetmap.org/wiki/Tag:junction%3Dyes) - Straßenknotenpunkt an sich"
also I couldn't find this string being used anyway.

Reply:
https://wiki.openstreetmap.org/wiki/DE:Tag:junction%3Dyes

![Screenshot_20230309-180019_Privacy Browser.png](https://user-images.githubusercontent.com/72605561/224100565-e3379fb2-beb1-403f-a102-e4c11cb0ecc6.png)

Reply:
https://wiki.openstreetmap.org/wiki/DE%3AKey%3Ajunction

![Screenshot_20230309-180214_Privacy Browser.png](https://user-images.githubusercontent.com/72605561/224101018-612c1dec-6419-49a0-8c25-f91a75e57409.png)